aapp_runner/read_aapp_config.py

```diff
@@ -245,92 +289,129 @@ def check_static_configuration(config):
     return True
 
 
-def read_config_file_options(filename, station, env, valid_config=None):
-    """
-    Read and checks config file
-    If ok, return configuration dictionary
-    """
+def load_config_from_file(filename):
+    """Load the yaml config from file, given the file-path"""
 
     # config = SafeConfigParser()
     import yaml
     with open(filename, 'r') as stream:
         try:
-            config = yaml.load(stream)
+            config = yaml.load(stream, Loader=yaml.FullLoader)
             import pprint
-            print(type(config))
-            pp = pprint.PrettyPrinter(indent=4)
-            pp.pprint(config)
+            # print(type(config))
+            #pp = pprint.PrettyPrinter(indent=4)
+            # pp.pprint(config)
```

Comment:
Should be removed

Reply:
Ok, it was not my code, so thats why I didn't dare before I understood what it was supposed to do. But @jkotro and @TAlonglong doesn't have the time atm so I remove for now, thanks!